istio/metadata.csv

```diff
@@ -140,6 +140,10 @@ istio.pilot.xds.push.timeout,count,,,,"Pilot push timeout, will retry.",-1,istio
 istio.pilot.xds.push.timeout_failures,count,,,,Pilot push timeout failures after repeated attempts.,-1,istio,
 istio.pilot.xds.pushes,count,,,,"Pilot build and send errors for lds, rds, cds and eds.",0,istio,
 istio.pilot.xds.write_timeout,count,,,,Pilot XDS response write timeouts.,0,istio,
+istio.pilot.xds.rds_reject,LastValue,,,,Pilot rejected RDS.,0,istio,
+istio.pilot.xds.lds_reject,LastValue,,,,Pilot rejected LDS.,0,istio,
+istio.pilot.xds.eds_reject,LastValue,,,,Pilot rejected EDS.,0,istio,
+istio.pilot.xds.cds_reject,LastValue,,,,Pilot rejected CDS.,0,istio,
```

Comment:
Here, where you have put `LastValue` you need to specify where the metric is a `count` or a `gauge`

Reply:
updated with `Count` for all